source/javascripts/lib/vendor/ui-utils/event.js

```diff
@@ -0,0 +1,30 @@
+/**
+* General-purpose Event binding. Bind any event not natively supported by Angular
+* Pass an object with keynames for events to ui-event
+* Allows $event object and $params object to be passed
+*
+* @example <input ui-event="{ focus : 'counter++', blur : 'someCallback()' }">
+* @example <input ui-event="{ myCustomEvent : 'myEventHandler($event, $params)'}">
+*
+* @param ui-event {string|object literal} The event to bind to as a string or a hash of events with their callbacks
+*/
+angular.module('ui.event',[]).directive('uiEvent', ['$parse',
```

Comment:
Exactly, the dropdown needs the ui.event module. This is normally part of the huge angular-ui library, but unfortunately you can't install it via bower and just include this single file.